### PR TITLE
add aqt_compile_eca endpoint

### PR DESCRIPTION
**Pull request by richrines1**

### CODE REVIEW

File: cirq_superstaq/service.py [hunk 1]
```diff
@@ -288,6 +288,40 @@ def aqt_compile(
...
         return cirq_superstaq.compiler_output.read_json_aqt(json_dict, circuits_is_list)
 
+    def aqt_compile_eca(
+        self,
+        circuit: cirq.Circuit,
+        num_equivalent_circuits: int,
+        random_seed: Optional[int] = None,
+        target: str = "keysight",
+    ) -> "cirq_superstaq.compiler_output.CompilerOutput":
+        """Compiles the given circuit to target AQT device with Equivalent Circuit Averaging (ECA).
```
Comment (vtomole) [inline cirq_superstaq/service.py:298]: Add link to https://arxiv.org/abs/2111.04572

**Review by singular-value — COMMENTED** [review]
lgtm

File: cirq_superstaq/service.py (hunk 1 above)
Comment (richrines1) [reply cirq_superstaq/service.py:298]: done.

File: cirq_superstaq/service.py [hunk 2]
```diff
@@ -288,6 +288,40 @@ def aqt_compile(
         )
         return cirq_superstaq.compiler_output.read_json_aqt(json_dict, circuits_is_list)
 
+    def aqt_compile_eca(
+        self,
+        circuit: cirq.Circuit,
+        num_equivalent_circuits: int,
+        random_seed: Optional[int] = None,
+        target: str = "keysight",
+    ) -> "cirq_superstaq.compiler_output.CompilerOutput":
```
[the remark this answers is not in the document]
Comment (richrines1) [reply cirq_superstaq/service.py:297]: filed #233

File: cirq_superstaq/service.py [hunk 3]
```diff
@@ -272,7 +272,7 @@ def aqt_compile(
         """Compiles the given circuit(s) to target AQT device, optimized to its native gate set.
 
         Args:
-            circuits: cirq Circuit(s) with operations on qubits 4 through 8.
+            circuits: cirq Circuit(s) to compile.
```
[the remark this answers is not in the document]
Comment (vtomole) [reply cirq_superstaq/service.py:275]: We've asserted that this is indeed true.

**Review by vtomole — APPROVED** [review]

**Pull request merged**